collect_app/src/main/java/org/odk/collect/android/provider/FormsProviderAPI.java

```diff
@@ -56,9 +56,6 @@ public static Uri getContentUri(String projectId) {
      * @deprecated This should be implemented as part of the UI/app code and should not be
```

Comment:
Remove comment

Reply:
This is still valid, I think?

Reply:
I guess the parts about where things should go are? I thought there was specific mention of constants that adds removed. But anyway, not very important.